src/features/tags/components/TagItem.js

```diff
@@ -1,24 +1,40 @@
 import React, { Component } from "react";
-import { NavLink } from "react-router-dom";
 import { connect } from "react-redux";
+import { NavLink } from 'react-router-dom';
 import { getTagById } from "../selectors";
+import { Link } from "@mui/material";
+import { ListItem, ListItemIcon, ListItemButton, ListItemText } from "@mui/material";
 
 class TagItem extends Component {
   render() {
     const { tag, key } = this.props;
-
+    
     return (
-      <li className="menu__list-item" name={tag.id} key={key}>
-        <NavLink
-          exact
+      <ListItem 
+        name={tag.id} 
+        key={key}
+        style={{
+          width: "100%",
+        }}
+      >
+        <Link 
+          name={tag.id}
+          key={key}
+          component={NavLink}
           to={`/tag/${tag.id}`}
-          activeClassName="menu__link menu__link--active active"
-          className="menu__link"
+          underline="none"
+          style={{
```

Comment:
These might be something to add in style, but in the MUI docs, the ListItem's background turns a light grey when hovered, and I'd like that for this feature too

Reply:
I wrapped the Link with a Button which gave the same effect. Is that alright, or should I change it?

Reply:
As long as it looks similar, I'm fine with it. When I tested locally, the background color wasn't changing